main.yml

```diff
@@ -21,7 +24,7 @@
   - debug: msg="{{ vm_ip.stdout }}"
 
   - name: ssh-copy-id to the Atomic host
-    shell: ssh-copy-id -i ~/.ssh/id_rsa.pub atomic-user@$"{{ vm_ip.stdout }}"
+    shell: ssh-copy-id -i /home/"{{ username }}"/.ssh/id_rsa.pub atomic-user@$"{{ vm_ip.stdout }}"
```

Comment:
$HOME would return `root` when I am root.
We can query with `uid=1000`, something like this - `cat /etc/passwd | grep $1000 | cut -d : -f 6`.
We could do `cat /etc/passwd | grep $USERNAME | cut -d : -f 6` instead..But base/atomic images do not have `ENV` var(s) `USERNAME` and `HOME` as normal user. They have it only for root user. 
The only way I can think of is to query with `uid=1000`. 
WDYT @gbraad ?